src/Test/Internal.elm

```diff
@@ -19,6 +19,14 @@ failNow record =
         (\_ _ -> [ Test.Expectation.fail record ])
 
 
+emptyDescriptionFailure : Test
+emptyDescriptionFailure =
+    failNow
+        { description = "This test has an empty description. Let's give it a nonempty one!"
```

Comment:
We do, actually - we're checking against `String.trim desc`, so "blank" seems appropriate! 👍